packages/components/src/index.js

```diff
@@ -29,6 +29,7 @@ export { default as CardFooter } from './card/footer';
 export { default as CardHeader } from './card/header';
 export { default as CardMedia } from './card/media';
 export { default as CheckboxControl } from './checkbox-control';
+export { default as CircularOptionPicker } from './circular-option-picker';
```

Comment:
Why is this (and the bar) exported here? Maybe instead oof exporting this low level component, we could instead add a generic duotone picker UI component here?

Reply:
The only reason I have it exported is because it seemed like it could be useful for others, and it was more convenient to export that than create a new `DuotonePalette` like is done for `ColorPalette`. So I'm guessing you'd prefer exporting a new `DuotonePalette` instead?

Reply:
Yeah, I prefer exporting high level components more than lower level ones as it matches what we did before.

Reply:
I started refactoring it, but I'm having second guesses if that's the right way to go about this. 

Creating a new `DuotonePalette` component in `@wordpress/components` would be splitting the duotone logic between `@wordpress/components` and `@wordpress/block-editor`. [`getGradientFromCSSColors`](https://github.com/WordPress/gutenberg/pull/26751/files#diff-6621fb4a69f1cfc8ce0f2ea9bf7a88fca448c8ccc4ddc5ae1fb43c56738b480bR52-R68) and [`getValuesFromColors`](https://github.com/WordPress/gutenberg/pull/26751/files#diff-6621fb4a69f1cfc8ce0f2ea9bf7a88fca448c8ccc4ddc5ae1fb43c56738b480bR88-R107) would need to be used in both packages. Otherwise, if I go ahead and abstract out those parts to keep the duotone functions in `@wordpress/block-editor`, I'm basically just left with a light wrapper around `CircularOptionPicker`.